### ISSUE TITLE
[v7d_transform] comp-start not working

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/libsim#43]
```
v7d_transform --input-format=BUFR --comp-stat-proc=254:2 --output-variable-list="B12101" --comp-step='1 00' --comp-start="2017-11-20 18:00" --output-format=csv  --comp-frac-valid=0 t2m.bufr tmax.csv
```
...creates a `tmax.csv` with no data. Changing the `comp-start` hour to "00:00" works.

(@dcesari already know the details and has the needed test data in /tmp/kalman)

**Comment by dcesari** [comment]
This turned out to be a very difficult task when working with time_definition=1 (verification time) as is the default in observation space. It can be solved by importing the volume with `--time-definition=0`, which is possible since last [commit](https://github.com/ARPA-SIMC/libsim/commit/324db1a6f0af1cd4b7da30fd00f47b7628449018), but this triggers a bug which will be solved tomorrow by merging #45.

**Comment by dcesari** [comment]
The bug has been solved after merging of #45, I keep it open because I have a partial universal solution which would allow it to work regardless of `time-definition`.

**Comment by brancomat** [comment]
Meanwhile: I confirm that the `--time-definition=0` thing is working with libsim 6.2.2
